utils/vscode/carbon.code-snippets

```diff
@@ -0,0 +1,55 @@
+{
+  "package": {
+    "scope": "carbon",
+    "prefix": "package",
+    "body": "package $fileNameWithoutExt",
+    "description": "Carbon package"
+  },
+  "Main Run function": {
+    "scope": "carbon",
+    "prefix": "Run",
+    "body": ["fn Run() -> i32 {", "\t$1", "\treturn 0;", "}"]
+  },
+  "New Function": {
+    "scope": "carbon",
+    "prefix": "fn",
+    "body": ["fn $1($2) ->  $3 {", "\t$4", "}"]
```

Comment:
This snippet seems like it could be kind of inconvenient if the user doesn't want the function to return anything (which will probably be pretty common).

Reply:
> This snippet seems like it could be kind of inconvenient if the user doesn't want the function to return anything (which will probably be pretty common).

If they don't want, they can remove the line it but I think having a default return in snippet would be nice